README.md

```diff
@@ -16,9 +16,9 @@ https://www.graalvm.org/latest/reference-manual/embed-languages/
 
 Download Maven or import as Maven project into your IDE.
 
-* `mvn package` build using javac
+* `mvn package` build using javac. Starting from GraalVM 24.1, you can use `mvn -Pisolated package` to build with the native isolate version of a guest language. By default, only the native isolate library for the current platform is installed. To install native isolate libraries for all supported platforms, use `mvn -Pisolated -Disolated.all.platforms package`.
 * `mvn test` to run the tests
-* `mvn exec:exec` to run the Main application
+* `mvn exec:exec` to run the Main application. Starting from GraalVM 24.1, you can use `mvn -Pisolated exec:exec` to utilize the native isolate version of a guest language.
```

Comment:
* `mvn exec:exec` to run the application. Starting from GraalVM Polyglot API version 24.1.0, you can use `mvn -Pisolated exec:exec` to embed the native isolate version of a guest language.

Reply:
fixed.